packages/util/src/environment.ts

```diff
@@ -16,6 +16,7 @@
  */
 
 import { CONSTANTS } from './constants';
+import { getDefaults } from './defaults';
```

Comment:
No need to change, just FYI this is creating a circular dependency which you may have noticed a warning about when you build this package. Storage also has some that we just live with so it seems to be fine. But if we have any problems in the future, we can probably just move `getGlobal()` (which is what `defaults.ts` is importing from here) into a separate file.